tests/fixtures/markdown/code.md

```diff
@@ -0,0 +1,8 @@
+Here is some Python:
+
+```python
```

Comment:
Yes, many people use py for shorthand anyway as it saves them some typing so I think adding that block anyway would be great.